src/utils/settings/index.js

```diff
@@ -139,6 +148,15 @@ function defaults(custom) {
   // `integrations` returns an array of valid integration items.
   withDefaults.integrations = integrations(withDefaults);
 
+  // validate push options
+  if (typeof withDefaults.streamingEnabled !== 'boolean') withDefaults.streamingEnabled = false;
+  if (withDefaults.streamingEnabled === true) {
```

Comment:
By this line it's a boolean already (if it wasn't, you assigned false on the previous line). Just go with `if (withDefaults.streamingEnabled)`